### ISSUE TITLE
picogl with npm

### ISSUE THREAD

**Issue by mreinstein** [tsherif/picogl.js#23]
@tsherif picogl is awesome! I'm interested in using this from node/browserify. Rather than including picogl from a script, I'd like to be able to do:

```javascript
var PicoGL = require('picogl')

var app = PicoGL.createApp(canvas)
```

I'm wondering if you'd be open to modularizing the code so that this usable from both npm and as a global script include. Happy to send a PR for this if you'd be interested!

great work on this library!

**Comment by tsherif** [comment]
Love this idea! I'm not too familiar with the whole require setup, so if you want to take a crack at it, go for it. As long as it doesn't add any dependencies and the script include usage still works, I'm all for it.

**Comment by mreinstein** [comment]
cool, I'll put together a PR tonight or tomorrow morning.

**Comment by tsherif** [comment]
Closed in #27

**Comment by mreinstein** [comment]
@tsherif in order for this to be usable on npm, you'll need to publish the package on the npm registry. You'll need an account on https://www.npmjs.com/
